lib/bolt/plugin.rb

```diff
@@ -83,7 +83,7 @@ def get_validated_task(task_name, params = nil)
           raise Bolt::Error.unknown_task(task_name) unless tasksig
 
           Bolt::Task::Run.validate_params(tasksig, params) if params
```

Comment:
Do we still need this line (or function) with these changes?

Reply:
Yes, this line validates that the parameters provided are acceptable by the task. It might make sense in the future to combine the logic on `Bolt::Task::Run` into `Bolt::Task` so this would be like:

```rb
task = Bolt::Task.from_task_signature(tasksig)
task.validate_params(params) if params
task
```

But for now I don't think it fits with this PR.